Gemfile

```diff
@@ -80,6 +80,7 @@ gem "descope", "~> 1.0"
 gem "invisible_captcha", "~> 2.3"
 gem "csv", "~> 3.3"
 gem "google-cloud-bigquery", "~> 1.50.0"
+gem "net-imap", "~> 0.5.6"
```

Comment:
Let me know if there is a better way to bump up the version of this gem.

This gem is currently pulled transitively as part of `actionmailer`, which hasn't been updated to bump this yet.